blockchain/validators/src/config.rs

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Config for validators.
+
+use crate::Error;
+use mc_blockchain_types::BlockIndex;
+use mc_crypto_keys::Ed25519Public;
+use serde::{Deserialize, Deserializer, Serialize};
+use std::{ffi::OsStr, fs, path::Path};
+
+/// A declaration that a pub key is valid for a specified range of block
+/// indexes.
+#[derive(Clone, Debug, Deserialize, Eq, Ord, PartialEq, PartialOrd, Serialize)]
+pub struct NodeKeyValidity {
+    /// The pub key, as a PEM file path or 64 hexadecimal characters.
```

Comment:
Would it be good to call out the path behaviour here?
Will the toml have any stand alone documentation?  
```suggestion
    /// The pub key, as a PEM file path or 64 hexadecimal characters.
    /// When providing a PEM file path, the path should be absolute or
    /// relative to the current working directory.
```